site/src/pages/release-notes/1.14.0.mdx

```diff
@@ -0,0 +1,345 @@
+- You no longer see an <type://EmptyEndpointGroupException> when an <type://EndpointGroup> is successfully
+  resolved with the initial <typeplural://Endpoint>. #3978
+- <type://HttpRequestBuilder#header(CharSequence,Object)> does not overwrite an old header value 
+  anymore. #3932 #3941
+  - A new header value will be appended to the old value.
+- You no longer see `413 Request Entity Too Large` when a server receives a cleartext (h2c) upgrade request
+  larger than 16384 bytes. #3859 #3913
+  - The maximum allowed length of an upgrade request will respect <type://ServerBuilder#maxRequestLength(long)>
+    or <type://VirtualHost#maxRequestLength()>.
+- Fixed <type://Multipart> decoder to correctly publish chunks one by one when streaming
+  a large <type://BodyPart>. #3774 #3783
+- You can now use `google.protobuf.{Struct, Value, ListValue, Any}` messages from your `proto` files when
+  using gRPC-JSON transcoder. #3986 #3992
+- <type://GrpcService> now automatically disables gRPC-JSON serialization for Protobuf 2 services
+  rather than throwing an exception. #4020 #4033
+- You no longer see `InvocationTargetException` with a Kotlin method whose return type is `Nothing`
+  in an annotated service. #3961 #4005
+- Actuator now correctly shows `Liveness` and `Readiness` of a Kubernetes instance. #3031 #3997
+
+## 🏚️ Deprecations
+
+- <type://CircuitBreaker#canRequest()> has been deprecated in favor of 
+  <type://CircuitBreaker#tryRequest()>. #4012
+- <type://Cookie#builder(String,String)> and <type://Cookie#of(String,String)> have been deprecated in favor of
+  <type://Cookie#secureBuilder(String,String)> and <type://Cookie#ofSecure(String,String)>. #3788 #3939
+- <type://GrpcServiceBuilder#setMaxInboundMessageSizeBytes(int)> and <type://GrpcServiceBuilder#setMaxOutboundMessageSizeBytes(int)>
+  have been deprecated in favor of <type://GrpcServiceBuilder#maxRequestMessageLength(int)> and 
+  <type://GrpcServiceBuilder#maxResponseMessageLength(int)>. #3999
+- <type://AbstractBlockingHttpVfs#get(Executor,String,Clock,String,HttpHeaders)?full> has been deprecated in favor of
+  <type://AbstractBlockingHttpVfs#get(Executor,String,Clock,String,HttpHeaders,MediaTypeResolver)?full>. #4009
+- <type://HttpVfs#get(Executor,String,Clock,String,HttpHeaders)?full> has been deprecated in favor of
+  <type://HttpVfs#get(Executor,String,Clock,String,HttpHeaders,MediaTypeResolver)?full>. #4009
+
+## ☢️ Breaking changes
+
+- <type://HealthChecker#of(Supplier,Duration)?full> and 
+  <type://HealthChecker#of(Supplier,Duration,EventExecutor)?full> now returns 
+  <type://ListenableHealthChecker> instead of <type://HealthChecker>. #4017
+- <type://ServerErrorHandler#renderStatus(ServiceConfig,RequestHeaders,HttpStatus,String,Throwable)> now 
+  requires an additional parameter <type://RequestHeaders>. #4037
+  - This allows a user to generate a different error response depending on the different request header value.
```

Comment:
Sorry about my previous review, the double `different` wording doesn't sound natural 

```suggestion
  - This allows a user to generate a different error response depending on the request header value.
```